Blocks/SS/idx_ts.h

```diff
@@ -48,9 +49,38 @@ CURRENT_STRUCT(IndexAndTimestamp) {
   CURRENT_CONSTRUCTOR(IndexAndTimestamp)(uint64_t index, std::chrono::microseconds us) : index(index), us(us) {}
 };
 
+// The same as `IndexAndTimestamp`, but with optional index member.
```

Comment:
Pls add a comment re. the value, not the meaning. Along the lines of: The timestamp is either the timestamp of the last entry published, or of the last call to `UpdateHead()`. In the former case, `index` is set to the index of the last published entry, and in the latter case `index` is unset.